api_practice/tokens.py

```diff
@@ -0,0 +1,14 @@
+from jose import JWTError, jwt
```

Comment:
move all authentication related classes to its own package called - "authentication". This package should be at the same level as "routers".